mijia-homie/src/main.rs

```diff
@@ -120,6 +120,8 @@ enum ConnectionStatus {
     MarkedDisconnected,
     /// Connected and subscribed to updates
     Connected,
+    /// Connected on a different Bluetooth adapter, on a system which has more than one.
+    ConnectedOnOtherAdapter,
 }
 
 #[derive(Debug, Clone)]
```

Comment:
I can believe that your approach will *work*, but I think it would be a better model of reality to make `Sensor::id` plural somehow. I realise that this might be a bunch more work though, so I'm happy to merge this as an intermediate step. Probably best to sleep on it.

Reply:
Pushed a first cut at this, with some TODOs still. We might want to change the interface of the `mijia` crate too.